test/shared/src/main/scala/zio/test/TimeVariants.scala

```diff
@@ -154,12 +155,18 @@ trait TimeVariants {
   /**
    * A generator of `java.time.ZoneId` values. Doesn't have any shrinking.
    */
-  final def anyZoneId: Gen[Random, ZoneId] = {
-    val zoneIds = ZoneId.getAvailableZoneIds().asScala
-    val idMap   = Set.range(0, zoneIds.size).zip(zoneIds).toMap
-
-    Gen.int(0, zoneIds.size).noShrink.map(idx => ZoneId.of(idMap(idx)))
-  }
+  final def anyZoneId: Gen[Random, ZoneId] =
+    Gen(
+      ZStream
+        .fromJavaIteratorTotal(ZoneId.getAvailableZoneIds.iterator)
+        .mapChunksM { ids =>
+          for {
+            random   <- ZIO.service[Random.Service]
+            shuffled <- random.shuffle(ids)
```

Comment:
Hmm, not sure how fast it is

Reply:
I think you can just use `Gen.elements` here as well once you get the available `ZoneId` values.

Reply:
Yeah, but I would need to convert those `ZoneId` values into Scala collection types such as `List` for instance and Scala `CollectionConverters` differ from version to version (at least, they are in different packages if we compare Scala 2.11 and Scala 2.13). 
That's why I decided to take this approach to avoid conversion between Java collections and Scala collections entirely. Otherwise that would make sense to use `Gen.elements`)

Reply:
Do I need to implement anyZoneId for each Scala version (in order to use appropriate CollectionConverters) and remove it from share?